package.json

```diff
@@ -52,7 +52,7 @@
     "clean-trace-jaeger": "node scripts/rm.mjs test/integration/basic/.next && TRACE_TARGET=JAEGER pnpm next build test/integration/basic",
     "debug": "cross-env NEXT_TELEMETRY_DISABLED=1 node --inspect packages/next/dist/bin/next",
     "postinstall": "git config index.skipHash false && node scripts/install-native.mjs",
-    "version": "npx pnpm@7.24.3 install --no-frozen-lockfile && IS_PUBLISH=yes ./scripts/check-pre-compiled.sh && git add .",
+    "version": "pnpm install --no-frozen-lockfile && IS_PUBLISH=yes ./scripts/check-pre-compiled.sh && git add .",
```

Comment:
Where does this `version` script run?

Reply:
I have no idea, but the engines section below makes sure it has to be version 8, otherwise pnpm itself will throw an error 🙃

Reply:
Indeed, as long as the job that is invoking that script calls `corepack enable` first 😄

Reply:
I'm assuming pnpm will look at the engines section regardless, that's independent of corepack

Reply:
Yeah I guess we'll see an error if `corepack enable` was not run before that script:

> During local development, pnpm will always fail with an error message if its version does not match the one specified in the engines field.

https://pnpm.io/package_json#engines